libcnb-test/src/test_runner.rs

```diff
@@ -56,6 +56,8 @@ impl TestRunner {
         config: C,
         f: F,
     ) {
+        check_required_pack_version();
```

Comment:
I think we should drop the Pack CLI version check, since:
- there's no way to only run it once before all tests (due to the way Cargo integration tests work), so we have to run it before every test
- our CI is already using Pack CLI 0.30 (thanks to https://github.com/buildpacks/github-actions already having updated the default version to it)
- this just leaves buildpack maintainers who will need to update - and realistically just the languages team at the moment (in the future libcnb-test might be used elsewhere, but by then this 0.30+ version check will be redundant and removed regardless)
- since the bug only affects this new feature (meta-buildpack support), we could just mention the pack 0.30+ requirement in the changelog entry for the new feature (+ also the rustdocs for `BuildpackReference::LibCnbRs`)